builtInFunctions/migrateUserName_test.go

```diff
@@ -0,0 +1,93 @@
+package builtInFunctions
+
+import (
+	vmcommon "github.com/multiversx/mx-chain-vm-common-go"
+	"github.com/multiversx/mx-chain-vm-common-go/mock"
+	"github.com/stretchr/testify/require"
+	"math/big"
+	"testing"
+)
+
+func TestNewMigrateUserNameFunc(t *testing.T) {
+	m, err := NewMigrateUserNameFunc(0, nil, nil)
+	require.Equal(t, err, ErrNilDnsAddresses)
+
+	m, err = NewMigrateUserNameFunc(0, make(map[string]struct{}), nil)
+	require.Equal(t, err, ErrNilEnableEpochsHandler)
+
+	dnsAddr := []byte("DNS")
+	mapDnsAddresses := make(map[string]struct{})
+	mapDnsAddresses[string(dnsAddr)] = struct{}{}
+	m, err = NewMigrateUserNameFunc(0, mapDnsAddresses, &mock.EnableEpochsHandlerStub{})
+	require.Nil(t, err)
+	require.False(t, m.IsInterfaceNil())
+
+	m.SetNewGasConfig(nil)
+	m.SetNewGasConfig(&vmcommon.GasCost{BuiltInCost: vmcommon.BuiltInCost{SaveUserName: 10}})
+	require.Equal(t, m.gasCost, uint64(10))
```

Comment:
Perhaps add a separate unit test for this code?
Or just rename the test func to smth like `TestNewMigrateUserNameFunc_SetNewGasConfig`? (although, I think the first solution would be cleaner)

Reply:
No need - did only for 100% code coverage.